website/sidebars.json

```diff
@@ -5,8 +5,12 @@
     "combined_pdf",
     {
       "type": "category",
-      "label": "Quickstart guide",
-      "items": ["quickstart_web", "quickstart_desktop", "quickstart_admin"]
+      "label": "Quickstart guides",
+      "items": [
+        "web_cli_install",
```

Comment:
I don't think we need to rename these files IMO, there are already various links that are not on our website that would break from this

Reply:
Good point, will change back